src/codegen/llvm.zig

```diff
@@ -4528,7 +4539,7 @@ pub const Object = struct {
 
         var attributes: Builder.FunctionAttributes.Wip = .{};
         defer attributes.deinit(&o.builder);
-        try o.addCommonFnAttributes(&attributes, zcu.root_mod);
+        try o.addCommonFnAttributes(&attributes, zcu.root_mod, false);
```

Comment:
Sorry for the nitpick but since I had some other comments already I might as well make this suggestion too:

```suggestion
        try o.addCommonFnAttributes(&attributes, zcu.root_mod, owner_mod.omit_frame_pointer);
```

(along with the following edits)